clojure-north.org

```diff
@@ -0,0 +1,360 @@
+* Themes
+
+ * Literate programming
+   * Background
+   * Org-Mode and Babel
+   * Quick simple clojure project example/template.
+   * References
+ * Building streams apps
+   * Assumptions
+   * References
+     * Kafka, Kafka Streams, Jackdaw
+   * Unit testing should be all you need
+     * Using transducers
+   * Unfortunately thats almost never the case
+     * Test machine to the rescue
+        * Integration testing (app behavior)
+        * System testing (multi app behavior) 
+ * SBA connector
+   * Show simplified example of the app using Literate Programming
+   * Caveats
+ * Logging, observability and monitoring
+   * Structured logging
+   * Realtime monitoring and instrumentation
+   * Exception handling
+     
+* Objectives
+
+ * Identify when Literate Programming is a good fit
+ * Use Jackdaw to create production ready Kafka streams appps
+ * Leverage best practices early on for faster turanound of Kafka Streams apps 
+ * Showcase how we implemnented a production Streaming app in weeks with Jackdaw
+
+* Script
+** Introduction
+
+  Hello! My name is Francisco but I go as Paco. I work at Funding Circle and
+  have been there for almost 3 years. One of the things that most interested
+  me at the time I joined was to learn and work with Kafka. I'm also a
```

Comment:
I'd probably reword the "one of the things..." sentence. Reading out loud and something about it is throwing me off. 

Maybe "One of the reasons I joined Funding Circle was the opportunity to learn about and work with Kafka"